osu.Game/Scoring/ScoreManager.cs

```diff
@@ -57,7 +57,7 @@ protected override ScoreInfo CreateModel(ArchiveReader archive)
         }
 
         protected override IEnumerable<string> GetStableImportPaths(Storage stableStorage)
-            => stableStorage.GetFiles(ImportFromStablePath).Where(p => HandledExtensions.Any(ext => Path.GetExtension(p)?.Equals(ext, StringComparison.OrdinalIgnoreCase) ?? false));
+            => stableStorage.GetFiles(".").Where(p => HandledExtensions.Any(ext => Path.GetExtension(p)?.Equals(ext, StringComparison.OrdinalIgnoreCase) ?? false));
```

Comment:
this looks like a regression?

Reply:
Actually it isn't.
`ArchiveModelManager` now operates on a storage pointing directly to the target data directory (`ImportFromStablePath`) instead of a Storage pointing to the root of the osu! stable installation
thus the use of "." to enumerate all directories / files in the current directory.

See this line:
	https://github.com/Game4all/osu/blob/a0fb308e8858265654c44e37b9ee3ba22b26c8e8/osu.Game/Database/ArchiveModelManager.cs#L592

Reply:
check the code again. test it. it's a regression.